.nuspec/Xamarin.Forms.nuspec

```diff
@@ -229,6 +229,7 @@
     <file src="..\Xamarin.Forms.Platform.UAP\bin\$Configuration$\Shell\ShellStyles.xbf" target="lib\uap10.0\Xamarin.Forms.Platform.UAP\Shell" />
     <file src="..\Xamarin.Forms.Platform.UAP\bin\$Configuration$\PickerStyle.xbf" target="lib\uap10.0\Xamarin.Forms.Platform.UAP" />
     <file src="..\Xamarin.Forms.Platform.UAP\bin\$Configuration$\Microsoft.UI.Xaml\Themes\generic.xbf" target="lib\uap10.0\Xamarin.Forms.Platform.UAP\Microsoft.UI.Xaml\Themes\" />
+    <file src="..\Xamarin.Forms.Platform.UAP\bin\$Configuration$\PromptDialog.xbf" target="lib\uap10.0\Xamarin.Forms.Platform.UAP" />
```

Comment:
@samhouts I believe I updated the nuspec...to be honest,  I have no idea what I'm doing there...but I think I made a good guess based on other files from the UWP platform...would you please double check?  Thanks!

Reply:
@bmacombe can you rebase this PR? The latest master doesn't require any of these to be specified on the nuspec

Reply:
@PureWeen Sure, I'll try to do that in the next few days

Reply:
@PureWeen After a rebase I'm getting this error when trying to compile

error MSB4184: The expression "[System.Version]::Parse('')" cannot be evaluated. Version string portion was too short or too long.

Any thoughts?

Reply:
delete your obj/bin directories and then you should be fine

Reply:
@PureWeen I did that a couple times and I'm down to this error on XF.Core.

Severity	Code	Description	Project	File	Line	Suppression State
Error	MSB4062	The "XFCorePostProcessor.Tasks.FixXFCoreAssembly" task could not be loaded from the assembly C:\git\Xamarin.Forms\Xamarin.Forms.Core\..\XFCorePostProcessor.Tasks\bin\Debug\net461\XFCorePostProcessor.Tasks.dll. Could not load file or assembly 'file:///C:\git\Xamarin.Forms\XFCorePostProcessor.Tasks\bin\Debug\net461\XFCorePostProcessor.Tasks.dll' or one of its dependencies. The system cannot find the file specified. Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask.	Xamarin.Forms.Core	C:\git\Xamarin.Forms\Xamarin.Forms.Core\Xamarin.Forms.Core.csproj	33

Reply:
your best bet will probably be to cleanout the directory with *git clean -xdf* 

keep in mind that anything you haven't checked in will get deleted

Reply:
@PureWeen That seemed to do the trick...thanks for that tip, I didn't know that one.  Rebase pushing shortly.

Reply:
Yay! We've been going through some pretty heavy csproj changes which unfortunately don't "just work" when they get merged in